auraed/src/runtime/cell_service/error.rs

```diff
@@ -54,7 +54,9 @@ impl From<CellsServiceError> for Status {
                 CellsError::FailedToAllocateCell { .. }
                 | CellsError::AbortedAllocateCell { .. }
                 | CellsError::FailedToKillCellChildren { .. }
-                | CellsError::FailedToFreeCell { .. } => Status::internal(msg),
+                | CellsError::FailedToFreeCell { .. }
+                | CellsError::CgroupIsNotACell { .. }
```

Comment:
are these internal errors? CgroupNotFound should probably be a `not_found` error. CgroupIsNotACell is probably a `bad_request`.

Reply:
I don't see a `bad_request`. I think `failed_precondition` might be the one (that's what we use for validation errors).